src/seed_data/factories/Note.factory.ts

```diff
@@ -0,0 +1,44 @@
+import { define } from "typeorm-seeding";
+import { Note } from "../../entity/Note";
+import { toUpperCase } from "../utils";
+
+import * as faker from 'faker';
+import * as _ from 'lodash';
+
+faker.seed(123);
+
+const tagOptions = createTagOptions();
+
+function createTagOptions() {
+    return _.times(10, function (n) {
+        return faker.lorem.word();
+    });
+}
+
+function chooseTags() {
+    const numTags = faker.random.number(3);
+    let tags = [];
+    var i: number;
+    for (i = 0; i < numTags; i++) {
+        const tagNum = faker.random.number(tagOptions.length - 1);
+        const tag = tagOptions[tagNum];
+        if (!tags.includes(tag)) {
+            tags.push(tag);
```

Comment:
i think the `faker.random.arrayElement` also offers a secondary parameter for number of elements to choose
From the faker docs,

><static> arrayElements(array, count)
takes an array and returns a subset with random elements of the array